CarePartner/Views/FollowedAccountsView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  FollowedAccountsView.swift
+//  CarePartner
+//
+//  Created by Pete Schwamb on 2/21/23.
+//
+
+import SwiftUI
+import TidepoolKit
+
+struct FollowedAccountsView: View {
+
+    @ObservedObject private var followedAccounts: FollowedAccounts
+    @ObservedObject private var client: TidepoolClient
+
+    @State private var showingAccountSettings = false
+
+    init(followedAccounts: FollowedAccounts, client: TidepoolClient) {
+        self.followedAccounts = followedAccounts
+        self.client = client
+    }
+
+    var body: some View {
+        // TODO: list of followed accounts with their summary views
+        ScrollView {
+            if followedAccounts.accounts.isEmpty {
+                Text("No accounts have shared data with you yet.")
+                    .padding(.horizontal)
+            } else {
+                ForEach(followedAccounts.accounts, id: \.name) { account in
```

Comment:
This will cause issues if a user is sharing with multiple people with the same name. Is there a unique ID field this can be keyed with? Or at the very least could `FollowedAccount` be `Hashable`?

Reply:
Yep, I'll be fleshing this out and there should be an account id in there soon.